contrib/stable/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# stable/Dockerfile
```

Comment:
Is the `stable` directory name some kind of shared convention across the projects? Looking just at the contents of `contrib`, it’s not clear what to expect in a subdirectory named `stable`.

Reply:
Yes, `stable`, `upstream` and `testing` are the directories that live under `contrib` on the other projects.  After this PR gets merged, I plan a follow up PR with Dockerfiles for those as well.  When I do that I'll add a README.md to contrib to provide a bit more clarity.